templates/ciena_saos_ssh_server_show_key.textfsm

```diff
@@ -0,0 +1,13 @@
+Value USERNAME (\S*)
```

Comment:
```suggestion
Value USERNAME (\S+)
Value KEY_STATUS (\S+)
```

Reply:
Do you mind addressing this and I think we're good to get this merged in.

Reply:
done.